frontend/src/components/Header/index.tsx

```diff
@@ -1,14 +1,24 @@
+import { Intent } from "@blueprintjs/core";
+import { Link } from "@reach/router";
 import React, { FC } from "react";
+import { ROUTES } from "src/common/constants/routes";
 import { HomepageLink } from "../common/HomepageLink";
 import AuthButtons from "./components/AuthButtons";
-import { MainWrapper, Wrapper } from "./style";
+import { MainWrapper, MyCollectionsButton, Right, Wrapper } from "./style";
 
 const Header: FC = () => {
   return (
     <Wrapper>
       <MainWrapper>
         <HomepageLink />
-        <AuthButtons />
+        <Right>
+          <Link to={ROUTES.MY_COLLECTIONS}>
```

Comment:
We actually need to conditionally render this with feature flag 🙏 Please help correct this! Thank you!

e.g., 

```tsx
import { get } from "src/common/featureFlags";
import { FEATURES } from "src/common/featureFlags/features";
import { BOOLEAN } from "src/common/localStorage/set";

{get(FEATURES.CREATE_COLLECTION) !== BOOLEAN.TRUE ? null : BUTTON_HERE }
```